cli/src/pcluster/aws/ec2.py

```diff
@@ -473,10 +473,15 @@ def describe_network_interfaces(self, network_interface_ids):
             self._paginate_results(self._client.describe_network_interfaces, NetworkInterfaceIds=network_interface_ids)
         )
 
+    @AWSExceptionHandler.handle_client_exception
+    def describe_volumes(self, volume_ids):
+        """Describe volumes."""
+        return list(self._paginate_results(self._client.describe_volumes, VolumeIds=[volume_ids]))
```

Comment:
if `volume_ids` is a list, the value passed to `VolumeIds` will be a list of list, which I think it's wrong.
A unit test would have caught this.

Reply:
Good catch!
I did this in a hurry to fix an issue with the mock_api and missed this.